xwiki-platform-core/xwiki-platform-ratings/xwiki-platform-ratings-api/src/main/java/org/xwiki/ratings/internal/SeparatePageRating.java

```diff
@@ -212,7 +212,7 @@ public void save() throws RatingsException
             document.setContentDirty(false);
             document.setCreatorReference(superadmin);
             document.setAuthorReference(superadmin);
-            context.getWiki().saveDocument(getDocument(), context);
+            context.getWiki().saveDocument(getDocument(), context.getMessageTool().get("rating.saveComment"), true, context);
```

Comment:
Well, IMO, the real problem is a design one. It should not be the Rating's task to save/delete itself, but it should be the RatingsManager's instead. Probably a bigger refactoring will be needed here at some point.